cosmic_ray/testing/test_runner.py

```diff
@@ -66,4 +68,4 @@ def __call__(self):
                                   test_result[1])
         except Exception as e:
```

Comment:
Maybe we can add pylint or something similar to analyze the code. If we do then removing the `as e` part will probably be flagged as warning. Although it will probably flag it as unused variable anyway. Removing the unused part for now.